### PR TITLE
Update atlas.json

### PR DESCRIPTION
**Pull request by Grymtn12**
Removed extra entries for rickroll qr code(originally there was 3 which wasn't added by any admin of our community).

Changed the description of the last one left to our preferred one. Also since the qr code is already going to youtube link we have changed link in to our community discord invite link.

One of the removed entries was from another subreddit(r/placeQRcode) which didn't belong to us. We dont have a subreddit.

### CODE REVIEW

**Comment by Codixer** [comment]
Resolve the conflict

**Comment by Grymtn12** [comment]
How do i actually resolve this, only thing i did is edit the atlas.json file and upload it, i removed the two lines that was saying rickrollqr and i edited the last one left, sorry im new to github

**Comment by AnonymousRandomPerson** [comment]
If you're not familiar with resolving merge conflicts and looking up tutorials doesn't help, a more crude option is to delete/reclone the repo and redo your changes.

File: web/atlas.json [hunk 1]
```diff
@@ -3824,7 +3823,6 @@
...
 {"id": "tx0p7b", "submitted_by": "Vegaxys", "name": "Diamond shovel from Minecraft", "description": "A diamond shovel from the game Minecraft.", "website": "", "subreddit": "/r/Minecraft", "center": [1740.5, 348.5], "path": [[1746.5, 342.5], [1741.5, 342.5], [1731.5, 354.5], [1739.5, 354.5], [1749.5, 344.5]]},
 {"id": "tx0p6c", "submitted_by": "Virgilcore", "name": "Forgot to install CS:S", "description": "An ERROR model from games that run on the Source engine. The background is the MISSING TEXTURE from Source games. Most commonly associated with Garry's Mod.", "website": "", "subreddit": "/r/SourceEngine", "center": [1789.5, 229.5], "path": [[1773.5, 207.5], [1805.5, 207.5], [1805.5, 251.5], [1774.5, 251.5], [1774.5, 208.5], [1773.5, 208.5]]},
-{"id": "tx0p3o", "submitted_by": "AustinQ", "name": "Rick Roll QR Code", "description": "While it doesn't appear in this screencap, the Rick Roll QR Code was rebuilt here just a couple minutes before the end.", "website": "", "subreddit": "/r/placeQRcode", "center": [1887.5, 933.5], "path": [[1869.5, 917.5], [1905.5, 917.5], [1905.5, 948.5], [1869.5, 949.5], [1869.5, 917.5]]},
```
Comment (AnonymousRandomPerson) [inline web/atlas.json:3827]: I see discussion on r/placeQRcode about these coordinates. Even if your group did not communicate with them, it looks like they independently contributed. All parties should be credited, so keep that subreddit in the remaining non-duped entries.

**Review by AnonymousRandomPerson — CHANGES_REQUESTED** [review]
Address comments and merge conflicts.

**Comment by Grymtn12** [comment]
updated the json file to include the r/placeQRcode, didnt change any other entry about rickroll except for the top one to the said updated on first request

**Comment by AnonymousRandomPerson** [comment]
Your PR now shows a ton of changes. I don't think that's intended?

**Comment by Grymtn12** [comment]
it should be only one line thats changed maybe i copypasted wrong the id tharts changed is twny3k at 472 that line only.

**Pull request closed without merging**